rolling-shutter/collator/batch/batch.go

```diff
@@ -0,0 +1,53 @@
+package batch
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	txtypes "github.com/shutter-network/txtypes/types"
+)
+
+func NewPendingTx(signer txtypes.Signer, txBytes []byte) (*PendingTransaction, error) {
```

Comment:
They are populated later on (partial initialisation) - we can discuss initialisation values though.
Rather than setting it to nil (which leaves distinction between "set" and "not yet set" we could explicitly set it to 0,
but then it is not clear later on wether the value was set to 0 explicitly or wether it is not initialised yet.

Reply:
I guess the problem is we'd need the base fee in order to be able to populate them which we don't at the time we insert them into the pool... Probably better to leave them nil then.